### ISSUE TITLE
Always show AUR tab in Option dialogue and hints user about what to install

### ISSUE THREAD

**Issue by tysontan** [aarnt/octopi#351]
Octopi currently hides AUR tab in its Options dialogue if there is no Pacaur/Yaourt/Trizen exists in the system. This creates difficulty for new users to learn about Octopi and its AUR support. Tested with Ocotopi version 0.9.9.5 (Manjaro 17.1.12 KDE)

My suggestions is:
1) Always show AUR tab in Octopi's Option dialogue
2) Grey out when Pacaur/Yaourt/Trizen is unavailable
3) Hint: You must install either one of the following components to enable AUR support: Pacaur/Yaourt/Trizen

I believe this will greatly improve Octopi's AUR user experience.

**Comment by demmm** [comment]
Octopi is used by more then just Arch and it's derivatives....AUR does not exists for other distributions, so makes no sense at all to always show AUR.

**Comment by tysontan** [comment]
It doesn't much sense at all to me that while Octopi does have AUR functionality, it doesn't want to educate its userbase about its existence. 

I learned about Octopi's AUR support secrets very late, when I accidentally looked into its optional dependencies. I don't think we should expect our users to be all wizards who knows everything before they use Octopi. Distros that use pacman is not rare breed anymore and ones like Manjaro has a very large, yet less techy userbase. 

Take Pacmac for example which has AUR enabled by default. It saves so much confusion, but it's a GTK app. I do hope Octopi as a Qt equivalent can handle it as nicely.

This issue doesn't affect me anymore because I've already learned it (the hard way), but I'm reporting here to share my perceptive, in order to help Octopi become better. I'm not here to challenge you from the Arch and its derivatives' camp. I hope my sincere intention not being interpret as "just Arch and you make no sense".

Maybe a detection whether the current system is arch or manjaro can help?
